python/deployment/utils.py

```diff
@@ -0,0 +1,54 @@
+import os
+from dotenv import load_dotenv
+
+SOL_ELLIPTIC_CURVE_FILENAME = "EllipticCurve.sol"
+
+# If you don't use the docker container, make sure to change this parameter
+ENV_PATH = "/app/.env"
```

Comment:
it is not good idea to save thie path here, because it make the script very dependent on the environment that the script is running. Create new .env file in this localization and remove this dependency.
```suggestion
```